modules/deploy/partials/cloud/controlplane-api.adoc

```diff
@@ -0,0 +1,288 @@
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+:tag-serverless-regions: api:ROOT:cloud-api.adoc#tag--ServerlessRegionService
+:tag-serverless-clusters: api:ROOT:cloud-api.adoc#tag--ServerlessClusterService
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+You can also use the Cloud API with Redpanda Serverless. See the xref:deploy:deployment-option/cloud/serverless.adoc[Serverless] documentation to learn about the limited availability (LA) release, including cluster limits, trial credits, and supported features.
+
+For the full Cloud API reference documentation, see xref:api:ROOT:cloud-api.adoc[Redpanda Cloud API Reference].
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters, networks, and resource groups.
+
+The Control Plane API consists of the following endpoints:
+
+* pass:a,m[xref:{tag-clusters}[ClusterService\]]
+* pass:a,m[xref:{tag-networks}[NetworkService\]]
+* pass:a,m[xref:{tag-operations}[OperationService\]]
+* pass:a,m[xref:{tag-resource-groups}[ResourceGroupService\]]
+* ServerlessRegionService
+* ServerlessClusterService
+// * pass:a,m[xref:{tag-serverless-regions}[ServerlessRegionService\]]
+// * pass:a,m[xref:{tag-serverless-clusters}[ServerlessClusterService\]]
+
+== Long-running operations
+
+Some endpoints do not directly return the resource itself, but instead return an operation. The following is an example response of xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[Create Network] (`POST /networks`):
```

Comment:
This is a bit nit-picky so if it is trouble, ignore me. But I thought it might be better to use cluster as an example because the terminology is familiar to users of byoc, dedicated and serverless whereas network isn't familiar to serverless users.